cloud/pkg/common/client/client.go

```diff
@@ -56,6 +57,15 @@ func InitKubeEdgeClient(config *cloudcoreConfig.KubeAPIConfig) {
 		crdKubeConfig := rest.CopyConfig(kubeConfig)
 		crdKubeConfig.ContentType = runtime.ContentTypeJSON
 		crdClient = crdClientset.NewForConfigOrDie(crdKubeConfig)
+
+		authKubeConfig, err = clientcmd.BuildConfigFromFlags(kubeConfig.Host, "")
+		if err != nil {
+			klog.Errorf("Failed to build config, err: %v", err)
+			os.Exit(1)
+		}
+		authKubeConfig.CAData = kubeConfig.CAData
+		authKubeConfig.CAFile = kubeConfig.CAFile
```

Comment:
why not use `clientcmd.BuildConfigFromFlags(config.Master, config.KubeConfig)` to replace these lines

Reply:
`authKubeConfig` will not use `kubeconfig` of the cloud, because `kubeconfig` contains cert data and key data which can 
 obtain admin permission.